src/deadline/job_attachments/api/manifest.py

```diff
@@ -213,6 +213,15 @@ def _manifest_diff(
 
     output: ManifestDiff = ManifestDiff()
 
+    # Helper function to update output datastructure.
+    def process_output(status: FileStatus, path: str, output_diff: ManifestDiff):
```

Comment:
This file is a public interface, we should keep this helper private by prefixing `_process_output`.

Reply:
No need, this is a nested function in a function! So its not exposed.

Reply:
Ahh didn't notice it's nested. Nice!